.github/chainguard/clone-packages-advisories.sts.yaml

```diff
@@ -4,7 +4,7 @@
 issuer: https://token.actions.githubusercontent.com
 subject_pattern: repo:chainguard-dev/enterprise-advisories:ref:refs/heads/main
 claim_pattern:
-  workflow_ref: chainguard-dev/enterprise-advisories/.github/workflows/build-and-publish-osv.yaml@refs/heads/main
+  workflow_ref: chainguard-dev/enterprise-advisories/.github/workflows/build-and-publish-(osv|secdb).yaml@refs/heads/main
```

Comment:
```suggestion
  workflow_ref: chainguard-dev/enterprise-advisories/\.github/workflows/build-and-publish-(osv|secdb)\.yaml@refs/heads/main
```

It's a regex right?

Reply:
Yeah, you're right. I remember asking about this early on and it seemed like we were find with the fact that `.` matches anything? I didn't understand fully tbh. I like your suggestion though. I like being precise with these.